src/main.ts

```diff
@@ -2,10 +2,11 @@ import { argv } from "process";
 import * as fs_old_school from "fs";
 const fs = fs_old_school.promises;
 
-import { ReportData, ImplementationReport, Constants } from './lib/types';
-import { get_report_data, get_template } from "./lib/data";
+import { TestData, ReportData, ImplementationReport, Constants } from './lib/types';
+import { get_test_metadata, get_report_data, get_template } from "./lib/data";
```

Comment:
Super-nit: since lines 5 and 27 refer to test_data and TestData, and I assume all 3 references are related, can we call this function get_test_data?